File: Andromeda/Models/DocumentFilter.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class DocumentFilter
    {
        public long DocumentId { get; set; }
        public string DocumentLetter { get; set; }
        public long DocumentNumber { get; set; }
        public long DocumentNumberFrom { get; set; }
        public long DocumentNumberTo { get; set; }
        public int EntityId { get; set; }
        public EntityModel EntityData { get; set; }
        public DocumentTypeEnum DocumentType { get; set; }
        public DateTime DocumentDateAt { get; set; }
        public DateTime DocumentDateFrom { get; set; }
        public DateTime DocumentDateTo { get; set; }
        public DateTime DocumentPreparedAt { get; set; }
        public DateTime DocumentBilledAt { get; set; }
        public DateTime DocumentDeliveredAt { get; set; }
        public DateTime DocumentLastUpdateAt { get; set; }
        public DateTime DocumentLastUpdateFrom { get; set; }
        public DateTime DocumentLastUpdateTo { get; set; }
        public DocumentStatusEnum DocumentStatus { get; set; }
        public int DocumentLinesQty { get; set; }
        public int DocumentNumerator { get; set; }
    }
}

File: Angular/src/app/models/document/document-filter.ts
import { Entity } from "../entity/entity";

export class DocumentFilter{
    DocumentId: number = 0;
    DocumentNumber: string = '';
    EntityId: number = 0;
    DocumentDate: Date = new Date('1/1/0001 12:00:00 AM');
    DocumentLastUpdateAt: Date = new Date('1/1/0001 12:00:00 AM');
    DocumentStatus: number = 0;
    DocumentEndDate: Date = new Date('1/1/0001 12:00:00 AM');
}